global.json

```diff
@@ -66,6 +66,30 @@
         ]
       },
       {
+        "id": "pushstarter_project",
+        "priority": 1,
+        "name": "Simple Push Project",
+        "description": "An example Project using Push notification framework",
+        "type": "default",
+        "icon": "icon-plane",
+        "category": "Samples",
+        "screenshots": [],
+        "appTemplates": [
+          {
+            "id": "pushstarter_ios_app",
+            "name": "Simple Push iOS App",
+            "type": "client_native_ios",
+            "repoUrl": "git://github.com/corinnekrych/pushstarter-ios-app.git",
```

Comment:
don't you want this to point to your local fork, but put it into https://github.com/feedhenry-templates